tools/dataset.cpp

```diff
@@ -197,3 +266,62 @@ void ukv_graph_import(ukv_graph_import_t* c_ptr) {
     else if (ext == ".csv")
         import_csv(c, max_batch_size);
 }
+
+void ukv_graph_export(ukv_graph_export_t* c_ptr) {
+
+    ukv_graph_export_t& c = *c_ptr;
+
+    auto export_method = &export_parquet;
+
+    ////// Choosing a method will look like this //////
+    //
+    // auto ext = c.paths_extension;
+    // auto export_method = ext == ".parquet" //
```

Comment:
This won’t work, you are comparing pointers

Reply:
Fixed